cfme/tests/configure/test_tag.py

```diff
@@ -124,6 +124,30 @@ def test_map_tagging_crud(appliance, category, soft_assert):
                                           .format(map_tag_entity.label))
 
 
+@pytest.fixture
+def csv_tag_file(create_vm, category, tag):
+    file_name = 'test_import_' + fauxfactory.gen_alphanumeric(4) + ".csv"
```

Comment:
I highly recomment using tempfile for this:
https://docs.python.org/3.7/library/tempfile.html

Reply:
Done